src/commands/build-and-push-image.yml

```diff
@@ -134,6 +134,11 @@ parameters:
     type: string
     default: ""
 
+  build:
+    description: Flag to trigger just build and not push image.
```

Comment:
This is a confusingly named value - it doesn't describe the behaviour of when this is false. Normally, this is fine, but in this case it does cause the command to push the image. Description could also be improved a bit.